azext_iot/tests/test_iot_central_int.py

```diff
@@ -185,6 +188,33 @@ def test_central_validate_messages_success(self):
         assert "Successfully parsed 1 message(s)" in output
         assert "No errors detected" in output
 
+    @pytest.mark.skipif(
+        not APP_SCOPE_ID, reason="empty azext_iot_central_scope_id env var"
+    )
+    @pytest.mark.skipif(
+        not APP_PRIMARY_KEY, reason="empty azext_iot_central_primarykey env var"
+    )
+    def test_device_conect(self):
+        device_id = "testDevice"
+        device_primary_key = generate_device_key(
+            device_id=device_id, primarykey=APP_PRIMARY_KEY,
+        ).decode()
+
+        credentials = {
+            "idScope": APP_SCOPE_ID,
+            "symmetricKey": {"primaryKey": device_primary_key},
+        }
+        device_client = helpers.dps_connect_device(device_id, credentials)
+
+        self.cmd(
+            "iot central device show --app-id {} -d {}".format(APP_ID, device_id),
+            checks=[self.check("id", device_id)],
+        )
```

Comment:
Minor nit - can we try sending telemetry to Central and monitoring that events are making it there? Or check the device-status is registered?

Reply:
assert device_client.connected ensures device is connected